pkg/cloud/gen/main.go

```diff
@@ -837,7 +837,12 @@ func (g *{{.GCPWrapType}}) List(ctx context.Context, zone string, fl *filter.F,
 
 {{- if .KeyIsGlobal}}
 	klog.V(5).Infof("{{.GCPWrapType}}.List(%v, %v): projectID = %v, ck = %+v", ctx, fl, projectID, ck)
+{{- if not .IsNetworkServices }}
 	call := g.s.{{.GroupVersionTitle}}.{{.Service}}.List(projectID)
+{{- else}}
+	parent := fmt.Sprintf("projects/%s/locations/global", projectID)
```

Comment:
Do we need to handle regional vs global?

Reply:
In the future, yes. I've seen implementation of Regional APIs just today - I need to asses how much work that may  need.
Currently I followed the existing behavior for `NetworkServices.Insert`: https://github.com/GoogleCloudPlatform/k8s-cloud-provider/blob/v1.33.0/pkg/cloud/gen/main.go#L918-L919

Reply:
can we flip the conditional to avoid `not`

```
{{- if .IsNetworkServices}}

{{else}